### PR TITLE
Update JetBrains section in "Set up your environment"

### PR DESCRIPTION
**Pull request by moritzruth**
Starting with WebStorm 2021.3 (and the corresponding versions of other IDEs), the Deno plugin uses the LSP ([WEB-48625](https://youtrack.jetbrains.com/issue/WEB-48625)).

Additionally, it uses the output of `deno types` instead of the bundled file.

You may want to wait with merging until WebStorm 2021.3 is officially released, that's why I created the PR as a draft.

### CODE REVIEW

**Comment by bartlomieju** [comment]
Thanks @moritzruth, I agree we should wait on the release before landing this PR.

**Comment by bartlomieju** [comment]
New WebStorm is out: https://twitter.com/WebStormIDE/status/1465331758652694529 so we can now land this PR.

**Review by satyarohith — APPROVED** [review]
LGTM. Thanks,  @moritzruth!

**Pull request merged**
